apps/sms/views/shared/test/unit/utils_test.js

```diff
@@ -1153,6 +1153,157 @@ suite('Utils', function() {
     });
   });
 
+  suite('Utils.throttle', function() {
+    setup(function() {
+      this.sinon.useFakeTimers(1);
```

Comment:
Not sure why you give 1 as parameter here, is it necessary for your test? It seems not necessary to set the specific timestamp for your test.